app/views/posts/index.html.erb

```diff
@@ -0,0 +1,16 @@
+
+<% @posts.each do |post| %>
+<h1> <%= link_to post.title, post %> </h1>
+
+<div>
+<p> <%= post.content.truncate(300, omission: '...') %> </p> 
```

Comment:
Not sure how smart this function is, does it break only at word breaks? If not, is that something worth doing?

Reply:
After checking the documentation I found out if I replace the argument " omission: '...'  " with "  separator: ' '  "  then it will truncate at a natural break.

Reply:
`<%= post.content.truncate(300, separator: ' ') %>`